pkgs/applications/misc/gkrellm/default.nix

```diff
@@ -15,8 +15,6 @@
 , wrapGAppsHook
 }:
 
-with lib;
```

Comment:
there is a `optionals` below that needs to be changed to `lib.optionals` or `lib.optional`